.github/workflows/release.yml

```diff
@@ -0,0 +1,108 @@
+name: Release
+
+on:
+  push:
+    tags:
+      - '*'
+
+permissions:
+  contents: write
+
+jobs:
+  publish-gh-releases:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Release
+        uses: softprops/action-gh-release@v2
+        with:
+          token: ${{ secrets.GH_TOKEN }}
+      # We faced problem like: https://github.com/cli/cli/issues/6599
```

Comment:
the issue is marked as fixed, maybe this is not necessary anymore?

Reply:
I removed that part. We'll see if it works when this first runs.